tests/src/main/java/com/cryptape/cita/tests/DecodeTxExample.java

```diff
@@ -42,7 +42,7 @@ private static String createSampleTransaction() throws IOException {
         long validUtilBlock = TestUtil.getValidUtilBlock(service).longValue();
 
         Transaction transferTx = new Transaction(
-                payeeAddr, nonce, quotaToDeploy, validUtilBlock, version, chainId, "1", data);
+                payeeAddr, nonce, quotaToDeploy, validUtilBlock, version, chainId, "28", data);
```

Comment:
Maybe you can set 28 to a const and make the const name more meaningful.

Reply:
Just one test value， used only in this method. Add annotation of this defined value instead.